cloudsync/sync/state.py

```diff
@@ -1256,3 +1256,20 @@ def unconditionally_get_latest(self, ent, i):
             ent[i].path = info.path
             if ent.ignored == IgnoreReason.NONE and not ent[i].changed:
                 ent[i].changed = time.time()
+
+    def get_state_lookup(self, side: int) -> 'SyncStateLookup':
+        return SyncStateLookup(self, side)
```

Comment:
where's the `state` parameter?

Reply:
`get_state_lookup()` is a member function of `SyncState` so `self` is the state param.

could probably name some of these things better, I'm open to suggestions.

Reply:
ah right, sorry pre-coffee brain talking